.travis.yml

```diff
@@ -1,16 +1,16 @@
 language: ruby
 cache: bundler
-sudo: false
 before_install:
   - gem update --system
   - gem install bundler
 before_script:
   - mysql -e 'create database ranked_model_test;'
   - psql -c 'create database ranked_model_test;' -U postgres
 rvm:
-  - 2.4.4
-  - 2.5.1
-  - 2.6.4
+  - 2.4
+  - 2.5
+  - 2.6
+  - 2.7
```

Comment:
Travis should pick the latest available version they have available, this should reduce maintenance burden a bit.